sky/serve/core.py

```diff
@@ -669,15 +637,10 @@ def tail_logs(
             with ux_utils.print_exception_no_traceback():
                 raise ValueError('`replica_id` must be None when using '
                                  'target=CONTROLLER/LOAD_BALANCER.')
-    controller_status, handle = backend_utils.is_controller_up(
+    handle = backend_utils.is_controller_up(
         controller_type=controller_utils.Controllers.SKY_SERVE_CONTROLLER,
         stopped_message='No service is found.')
-    if handle is None or handle.head_ip is None:
-        msg = 'No service is found.'
```

Comment:
Double check all these cases? Without server controller:

This PR
```
» sky serve logs xx 1
```
(no outputs, just exit 1)

Master
```
» sky serve logs xx 1                                                     
No service is found.
```